CHANGELOG.md

```diff
@@ -4,6 +4,10 @@ This project uses [Semantic Versioning 2.0.0](http://semver.org/).
 
 ## main
 
+## 1.3.0
```

Comment:
If we add a version then we should also append `(Unreleased)` to the version so if the change ends up in main there is no confusion if a developer does not find a corresponding release/tag.

Reply:
Moved to main section, will update version during release